docs/articles/clustering/cluster-metrics.md

```diff
@@ -0,0 +1,391 @@
+---
+uid: cluster-metrics
+title: Akka.Cluster.Metrics module
+---
+
+# Akka.Cluster.Metrics module
+
+The member nodes of the cluster can collect system health metrics and publish that to other cluster nodes and 
+to the registered subscribers on the system event bus with the help of Cluster Metrics Extension.
+
+Cluster metrics information is primarily used for load-balancing routers, 
+and can also be used to implement advanced metrics-based node life cycles, such as “Node Let-it-crash” when CPU steal time becomes excessive.
+
+Cluster members with status `WeaklyUp`, if that feature is enabled, will participate in Cluster Metrics collection and dissemination.
+
+## Metrics Collector
+
+Metrics collection is delegated to an implementation of `Akka.Cluster.Metrics.IMetricsCollector`.
+
+Different collector implementations may provide different subsets of metrics published to the cluster. 
+Metrics currently supported are defined in `Akka.Cluster.Metrics.StandardMetrics` class:
+* `MemoryUsed` - total memory allocated to the currently running process
+* `MemoryAvailable` - memory, available for the process
+* `MaxMemoryRecommended` - if set, memory limit recommended for current process
+* `Processors` - number of available processors
+* `CpuProcessUsage` - CPU usage by current process
+* `CpuTotalUsage` - total CPU usage
+
+> Note: currently, due to some .NET Core limitations `CpuTotalUsage` is the same as `CpuProcessUsage` metrics, 
```

Comment:
I'd include a link to the Github issue where we discuss fixing this.

Reply:
Sure